src/lib/contentful.js

```diff
@@ -10,3 +10,12 @@ export const client = contentful.createClient({
 //   .getEntry('23ajyJGN6FZHErhMhAjYLu')
 //   .then((entry: any) => console.log(entry))
 //   .catch(console.error)
+
+// sample code for SSR
```

Comment:
do you think we can remove the commented code?

Reply:
removed commented code